src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] The objective of this milestone is to practice React class components. Kindly refactor the `App` component to a React class component. :pray:

Reply:
Thank you. It is done. The App is now a class component